Investigate how to fix the tests, when we upgrade to com.github.marschall:memoryfilesystem >= 2.1.0

# Task Description

We need to investigate what will need to be done in order to upgrade to `com.github.marschall:memoryfilesystem` from `2.1.0` to `0.9.1` due to incompatibilities with JDK11. This causes a problem with timestamps like this:
```
[ERROR] Failures:
[ERROR]   ReadAttributesTest.readAttributesAll:463 expected:<2020-08-23T10:19:52.0635Z> but was:<2020-08-23T21:19:52.635Z>
[ERROR]   ReadAttributesTest.readAttributesFile:92 expected:<2020-08-23T21:19:52.669127Z> but was:<2020-08-23T21:19:52.669Z>
```

This issue seems to be related to the higher precision of timestamps introduced in JDK 9 ([JDK-8068730](https://bugs.openjdk.java.net/browse/JDK-8068730))

# Task Relationships

This task:
* Is a follow-up of: #7 , #39 , #43

# Help

* Our [chat channel](https://chat.carlspring.org/channel/s3fs-nio2-community)
* Points of contact:
  * @carlspring
  * @steve-todorov
  * @sbespalov

--- a/pom.xml
+++ b/pom.xml
@@ -86,6 +86,7 @@
         <version.tika>1.25</version.tika>
         <version.findbugs.jsr305>3.0.2</version.findbugs.jsr305>
         <version.mockito>3.6.28</version.mockito>
+        <version.marschall.memoryfilesystem>2.1.0</version.marschall.memoryfilesystem>
     </properties>
 
     <build>
@@ -244,7 +245,7 @@
         <dependency>
             <groupId>com.github.marschall</groupId>
             <artifactId>memoryfilesystem</artifactId>
-            <version>0.9.1</version>
+            <version>${version.marschall.memoryfilesystem}</version>
             <scope>test</scope>
         </dependency>
 

